website/for_teachers.py

```diff
@@ -1201,7 +1203,9 @@ def create_adventure(self, user, class_id=None):
             "name": name,
             "classes": [class_id],
             "level": 1,
+            "levels": "1",
```

Comment:
levels is a list of level strings!
```suggestion
            "levels": ["1"],
```